src/main/java/picard/util/LiftoverUtils.java

```diff
@@ -209,6 +301,16 @@ protected static void leftAlignVariant(final VariantContextBuilder builder, fina
         int theStart = start;
         int theEnd = end;
 
+        final String refString = StringUtil.bytesToString(referenceSequence.getBases(), start-1, end - start + 1);
+        //make sure that referenceAllele matches reference
+        final Allele refAllele = alleles.stream().filter(Allele::isReference).findAny().orElseThrow(()->new RuntimeException("How did that happen? No reference allele?"));
+
```

Comment:
Do we really need this check here?  It seems more appropriate to put this check where the ref allele was created.

Reply:
hmmm. But that really doesn't fit with the flow of the original algorithm as the actual check against the reference happens at the end, before trying to add the VC...if you feel strongly about this, let's talk more in person.

Reply:
I don't feel strongly... it's just weird.  As a reviewer of the code it just looks out of place.
Can we move it up to the method that calls this?